spec/models/users/setting_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+
+RSpec.describe Users::Setting, type: :model do
+  describe "validations" do
+    subject { create(:setting) }
+
+    let(:user) { create(:user) }
+    let(:setting) { create(:setting, user: user) }
```

Comment:
Some of the errors you are getting on the tests i.e.

```
undefined method `user=' for #<Users::Setting:0x000000000c6f8978>
```
and
```
Validation failed: User can't be blank,
```
is implying that we are not specifying the user when we create a setting.

This is because we've set 
```
subject { create(:setting) }
```
without specifying a user. 

Hence we can amend it as such:
```suggestion
    let(:user) { create(:user) }
     subject { create(:setting, user: user) }
```
and remove the line:
```
 let(:setting) { create(:setting, user: user) }
```

In addition, when we use `described_class` in the rest of the file, we'd need to instantiate it, so in the describe blocks we can add
```
let (:setting) { described_class.new }
```
and rename `described_class` to `setting`